apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9957,12 +9957,15 @@ input.mat-input-element{
 .delete-modal-overlay {
     z-index: 9992; /*for delete custom zIndex modal*/
 }
-/* responsive */
-@media (max-width: 1440px) {
-    .font-xxlarge {
-        font-size: 20px;
+.custom-select{
```

Comment:
Do format this code block, please don't format the whole file as it will create conflicts

Reply:
Okay